src/ifp/README.md

```diff
@@ -52,30 +52,29 @@ die =  ( 0, 0 )
 
 ```tcl
 initialize_floorplan
+  -die_area {llx lly urx ury} | -utilization util
```

Comment:
The requirements are actually more complex.  You can either use (-utilization) or (-die_area and -core_area).

Reply:
I'm not sure why -utilization makes -core_space optional (0 by default) and -die_area requires -core_area.  It feels like both should either be required or not (probably required).  @rovinski @gadfort any opinions?

Reply:
I think die_area and core_area go together so if one is used the other should be required (it's possible to use core_space to derive the other if one wants to get fancy about it).
I would be okay saying core_space is required with utilization + aspect_ratio.

Reply:
(utilization && core_space) || (die_area && core_area)

Reply:
Look like a consensus.  Please make the change in this PR.

Reply:
Based on the above -core_space  should be mandatory not optional.  I don't see that reflected in your update.